spotlight-client/src/contentModels/Metric.ts

```diff
@@ -194,24 +220,59 @@ export default abstract class Metric<RecordFormat extends MetricRecord>
     return this.allRecords;
   }
 
+  get unknowns(): Unknowns | undefined {
+    return undefined;
+  }
+
+  generateReadme(): string {
```

Comment:
I think this should probably be a computed getter instead ... this happens to work fine at the moment because we only call it within the download method, but if we wanted to put this text somewhere in the UI again later it won't be properly observable

Reply:
Sounds good to me, also if we re-download the data for some reason it'll just be computed once

Reply:
that may not be completely true since the download function is not really a reactive context, but it's also not any worse than it was previously